Eastern-Research-Group/mywaterway
# Feature/137 update epa template

## Related Issues:
* [HMW-137](https://jira.epa.gov/browse/HMW-137)
* [HMW-652](https://jira.epa.gov/browse/HMW-652)

## Main Changes:
* Addressed issues with the EPA template mobile header.
  * These changes are identical to those made in LEW's [PR #139](https://github.com/Eastern-Research-Group/lew/pull/139), with the addition of a z-index specifier to ensure the drawer is on top of other page elements.

## Steps To Test:
1. Test the various mobile header components on the main page (`index.html`) and the error pages (`400.html` & `500.html`).
2. On the main page, ensure the mobile drawer menu is not partially blocked by other page components when open, e.g. the "How's My Waterway?" banner, the `.unsupportedMessageContainer`, or the glossary (it shouldn't be possible to have both the drawer and the glossary open simultaneously).
